adplugdb/getopt.c

```diff
@@ -101,7 +101,12 @@
    GNU application programs can use a third alternative mode in which
    they can distinguish the relative order of options and other arguments.  */
 
+#ifdef DJGPP
+#include "mygetopt.h"
```

Comment:
Is mygetopt.h provided by DJGPP, or is this file missing in the PR?

Reply:
Already part of Adplug, it's in the same folder, https://github.com/adplug/adplug/blob/master/adplugdb/mygetopt.h

Reply:
My mistake, I looked in the wrong directory